test/test_download.py

```diff
@@ -73,11 +73,16 @@ class TestDownload(unittest.TestCase):
 
     def __str__(self):
         """Identify each test with the `add_ie` attribute, if available"""
+
+        def strclass(cls):
+            """From python2.7/unittest/util.py; in 2.6 it was _strclass so we cannot import it."""
+            return "%s.%s" % (cls.__module__, cls.__name__)
+
         add_ie = getattr(self, self._testMethodName).add_ie
         if add_ie:
-            return "%s (%s) [%s]:" % (self._testMethodName, unittest.util.strclass(self.__class__), add_ie)
```

Comment:
Although, we could also just stop printing the class name, maybe that would be better?
I was going for the minimal change, but thinking about it some more, it's basically just noise:

```
test_Generic_10 (test.test_download.TestDownload):  ... ok
test_GenerationQuoi (test.test_download.TestDownload):  ... ERROR
test_Generic (test.test_download.TestDownload):  ... ok
test_Generic_101 (test.test_download.TestDownload):  ... FAIL
test_Generic_102 (test.test_download.TestDownload):  ... ok
test_Generic_103 (test.test_download.TestDownload) [TwentyMinuten]: ... ok
test_Generic_104 (test.test_download.TestDownload) [VideoPress]: ... ok
test_Gazeta_1 (test.test_download.TestDownload) [EaglePlatform]: ... FAIL
test_Generic_12 (test.test_download.TestDownload):  ... ok
test_Generic_13 (test.test_download.TestDownload):  ... ok
test_Generic_14 (test.test_download.TestDownload):  ... ok
test_Generic_11 (test.test_download.TestDownload):  ... ok
test_Generic_15 (test.test_download.TestDownload):  ... ok
```

could become:

```
test_Generic_1:  ... ok
test_Generic_10:  ... ok
test_GenerationQuoi:  ... ERROR
test_Generic:  ... ok
test_Generic_101:  ... FAIL
test_Generic_102:  ... ok
test_Generic_103 [TwentyMinuten]: ... ok
test_Generic_104 [VideoPress]: ... ok
test_Gazeta_1 [EaglePlatform]: ... FAIL
test_Generic_12:  ... ok
test_Generic_13:  ... ok
test_Generic_14:  ... ok
test_Generic_11:  ... ok
test_Generic_15:  ... ok
```

But I dunno, maybe someome finds the class name important for some reason?